Emby.Server.Implementations/Emby.Server.Implementations.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.4.3" />
```

Comment:
This should be moved to `Jellyfin.Server`

Reply:
Ok